app/components/ui/atoms/PersonPod.md

```diff
@@ -66,32 +67,28 @@ Selected icon:
 Disabled:
 
 ```js
-<PersonPod
-  name="Richard Aldrich"
+;<PersonPod
+  name="Brian B Aldrich"
   institution="Utrecht University"
-  keywords="cell biology"
+  keywords={['Cell Biology']}
   isKeywordClickable={false}
-  onKeywordClick={() => console.log('keyword clicked')}
-  status="Currently unavailable"
-  isStatusShown={true}
   iconType="add"
   isIconClickable={false}
-  onIconClick={() => console.log('icon clicked')}
 />
 ```
 
-### ChoosePersonPod
+### SelectButton
```

Comment:
Thanks :sweat_smile: that was bugging me.

Could we change the other line too (ChosenPersonPod)? Or maybe just remove it?